### DISCUSSION TITLE
404 error when changing label height

### DISCUSSION THREAD

**Discussion by ramaht** [porrey/Virtual-ZPL-Printer#19] [General]
Hi, am getting "Error: HTTP 404 Not Found" error when I change the height of a label to a value behind the comma in inches.
Tried changing to 2.5 inch (also applies to CM set to 10x6 or MM to 100x60)

Change it back to 3 inch, works fine.

**Comment by porrey** [comment]
I am unable to reproduce. Please provide details to reproduce and version.

**Comment by ramaht** [comment]
Hi, V2.3.0
press add / leave stock (host 127.0.0.1:9100) - inches - size 4" by 6" > save > start > test 2x2 Product label = succes
stop > edit > height 2.5 > save > start > test 2x2 Product label = Labelary Error     Error: HTTP 404 Not Found

Change the height to 2 , works fine.

No other settings changed.

**Reply by porrey** [reply]
I am still unable to reproduce.

<img width="919" alt="Screenshot 2023-01-13 at 6 12 34 PM" src="https://user-images.githubusercontent.com/12475115/212440206-67e7791b-70f3-4d3c-a624-c4d0a9c4c207.png">
